docs/reference/issuers.rst

```diff
@@ -50,6 +50,71 @@ namespaces, you should consider creating a :doc:`ClusterIssuer <clusterissuers>`
 resource. This is almost identical to the Issuer resource, however is
 non-namespaced and so it can be used to issue Certificates across all namespaces.
 
+***************************************
+Certificate Duration and Renewal Window
+***************************************
+
+Cert-manager's issuers and clusterissuers support custom certificate duration
+and renewal window.
+
+**Important**: The backend service implementation can choose to generate a
+certificate with a different validity period than what is requested in the
+issuer.
+
+The table below shows the support state of the different backend services used
+the issuer types:
+
+=======  ============================================================
+Issuer   Description
+=======  ============================================================
+ACME     The protocol supports it but it is currently not implemented
+         in Boulder (Let's Encrypt).
+CA       Fully supported.
+Vault    Fully supported. (Although the requested duration must be
+         lower than the configured Vault role's TTL)
+=======  ============================================================
+
+The table below shows the default duration and renewal window per
+issuer:
+
+======  =========================  =========================
+Issuer  Duration                   RenewBefore
+======  =========================  =========================
+ACME    Implementation dependent   Implementation dependent
```

Comment:
renewBefore is not something set by ACME (i.e. is not 'implementation dependent')